src/components/Contact.js

```diff
@@ -0,0 +1,53 @@
+const Contact = () => {
+    return ( 
+        <div className="container mt-5">
+
+  <h2 className="text-center "> Contact Us</h2>
+  
+  <div className="row">
+
+    <div className="col-md-4">
+
+      <img src="img/footer/getintouch.png" height="200px" alt="pizza"/>
```

Comment:
Move this in Contact folder in images directory.

Reply:
done

Reply:
👍